README.md

```diff
@@ -750,6 +750,50 @@ config.module
       .options({ presets: ['babel-preset-es2015'] });
 ```
 
+#### Config module rules use before
```

Comment:
The other sections use a naming of form "Config module rules uses (loaders): modifying options" - I don't have a preference as to which to use, but wanted to note that this wasn't consistent with them, in case this wasn't intentional.